elfo-logger/src/config.rs

```diff
@@ -12,6 +14,9 @@ pub(crate) struct Config {
     #[serde(default)]
     pub(crate) format: Format,
 
+    #[serde(default)]
+    pub(crate) max_line_size: Option<MaxLineSize>,
```

Comment:
Yeah, the `_line_` word here used to describe limit for each line in the `buffer`, this limit applies to each line created by `CurrentLine` in the `LineBuffer`. `max_size` in that case would be wrong, since it makes this meaning even more unobvious.